core/src/main/java/discord4j/core/util/Mentions.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of Discord4J.
+ *
+ * Discord4J is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Discord4J is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Discord4J.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package discord4j.core.util;
+
+import discord4j.common.util.Snowflake;
+import discord4j.core.object.entity.Role;
+import discord4j.core.object.entity.User;
+import discord4j.core.object.entity.channel.Channel;
+import discord4j.rest.util.Permission;
+
+public final class Mentions {
+    /**
+     * The <i>raw</i> mention for notifying all users in a channel.
+     *
+     * @see Permission#MENTION_EVERYONE
```

Comment:
the mentions in message require declare the bot/message can use mentions not? if so then maybe need mention this too?

Reply:
Sorry, I don't understand the question?

Reply:
sorry, i mean can be mentioned in docs about how mentions works with the "allowed mentions" based in https://discord.com/developers/docs/resources/channel#allowed-mentions-object this and the permission affect the use of the mentions to roles, user, everyone.

Reply:
I pushed some documentation changes - how does that look?

Reply:
> I pushed some documentation changes - how does that look?

looks good, also need to be put in the role/user method because this has the same behaviour.